ParcelDistributionCenter/ParcelDistributionCenter.Web/Controllers/PackagesController.cs

```diff
@@ -61,6 +62,34 @@ public ActionResult DeletePackage(int packageNumber)
             TempData["MessageClass"] = "alert-danger";
             return RedirectToAction(nameof(DisplayPackages));
         }
+      
+        // GET: ProductsController/Delete/5
+        public ActionResult Delete(int id)
```

Comment:
Czemu w sumie stworzyłeś nową metodę Delete? Wydaje mi się, że już istnieje metoda usuwająca paczkę. Musimy się więc zastanowić, czy robimy osobne okienko do usuwania paczki, czy zostawiamy to stare :P Może i lepiej nawet z okienkiem

Reply:
Zrobiłem oddzielną metodę bo ta zrobiona wcześniej do końca mi nie pasowała. Ta którą ja zrobiłem jest bardziej uniwersalna i dzięki temu użyłem jej w kilku miejscach.